src/medCore/gui/toolboxes/medDiffusionAbstractToolBox.h

```diff
@@ -23,8 +23,8 @@ class MEDCORE_EXPORT medDiffusionAbstractToolBox : public medToolBox
 {
     Q_OBJECT
 public:
-    medDiffusionAbstractToolBox(QWidget *parentToolBox = 0);
-    virtual ~medDiffusionAbstractToolBox();
+    medDiffusionAbstractToolBox(QWidget *parentToolBox = 0){}
+    virtual ~medDiffusionAbstractToolBox(){}
```

Comment:
The constructor should pass the parent widget on to the base constructor.
The destructor can be removed if it does nothing.
Same for below.